makefiles/vars.inc.mk

```diff
@@ -8,7 +8,7 @@ export QUIET                 # The parameter to use whether to show verbose make
 export APPLICATION           # The application, set in the Makefile which is run by the user.
 export APPLICATION_MODULE    # The application module name.
 export BOARD                 # The board to compile the application for.
-export CPU                   # The CPU, set by the board's Makefile.include.
+export CPU                   # The CPU, set by the board's Makefile.features.
```

Comment:
I like the comment specifying where CPU should be defined, it documents a little bit the build system variables. Could we do the same for CPU_MODEL?

Reply:
I do it now.